PacketDotNet/WakeOnLanPacket.cs

```diff
@@ -129,6 +133,35 @@ public PhysicalAddress DestinationAddress
             }
         }
 
+        /// <summary>
+        /// The Password field is optional, but if present, contains either 4 bytes or 6 bytes.
+        /// If a 4-byte password is present, it will be dissected as an IPv4 address 
+        /// and if a 6-byte password is present, it will be dissected as an Ethernet address.
+        /// </summary>
+        public string Password
```

Comment:
I think this should just return a byte array as to my understanding this doesn't need to be a mac (although it can be interpreted as such).
Furthermore, base CLR types are named as String. 
Linq isn't needed here either.

Reply:
108 / 106 are magic numbers, where are they from and what do they declare? Can you add a test for this? 
What about a setter?

Reply:
As you mentioned the Password is returned a byte[].
Test also has been added.